.gitattributes

```diff
@@ -0,0 +1 @@
+alternate_formats/* -diff
```

Comment:
This was attempting to make the diffs of these files not show up, or not be as prominent at least, during local git commands.  

So you see this when doing a git diff, instead of the whole diff:
```
diff --git a/alternate_formats/legislators-current.json b/alternate_formats/legislators-current.json
index 9faeb22..956ccae 100644
Binary files a/alternate_formats/legislators-current.json and b/alternate_formats/legislators-current.json differ
```